src/vcf/fixer.cpp

```diff
@@ -14,398 +14,585 @@
  * limitations under the License.
  */
 
+#include <set>
+
 #include "vcf/fixer.hpp"
 #include "vcf/string_constants.hpp"
 
 namespace ebi
 {
-    namespace vcf
-    {
-        void Fixer::fix(size_t line_number, std::vector<char> &line, Error &error)
-        {
-            this->line_number = line_number;
-            this->line = &line; // this will be valid until this function returns
-            error.apply_visitor(*this);
-            this->line = nullptr;
-        }
+  namespace vcf
```

Comment:
I agree on doing a specific PR for silly maintenance. removing the indentation commit from this PR should be trivial